core/store/models/run_output.go

```diff
@@ -0,0 +1,119 @@
+package models
+
+import (
+	"fmt"
+
+	"github.com/tidwall/gjson"
+)
+
+// RunOutput represents the result of performing a Task
```

Comment:
What is a Task? Are they the TaskRun structs?

Reply:
Yeah, a Job Spec specifies a list of Tasks to walk through, ie Task Specifications. Time a Job is run a TaskRun is created for each Task.